ts/Core/Renderer/SVG/SVGRenderer.ts

```diff
@@ -37,6 +37,7 @@ import SVGLabel from './SVGLabel.js';
 import AST from '../HTML/AST.js';
 import TextBuilder from './TextBuilder.js';
 import U from '../../Utilities.js';
+import options from '../../../Accessibility/Options/Options';
```

Comment:
This is not intended, right?

Reply:
Woops, must've snuck in there from an auto import